cmd/job_run.go

```diff
@@ -35,6 +35,12 @@ func JobRunFactory(ui cli.Ui) cli.CommandFactory {
 	}
 }
 
+//isPathAbsUnix checks if a path is absolute on a Unix system.
+//Derived from https://github.com/golang/go/blob/1106512db54fc2736c7a9a67dd553fc9e1fca742/src/path/filepath/path_unix.go#L12
```

Comment:
Move this to the svc package as it is a validation criteria for our implementation: check this out: https://godoc.org/gopkg.in/go-playground/validator.v9